keyboards/handwired/tractyl_manuform/5x6_right/rpi2040/info.json

```diff
@@ -0,0 +1,102 @@
+{
+    "keyboard_name": "Tractyl Manuform (5x6) RP2040",
+    "url": "https://github.com/Schievel1/baby_got_track",
+    "maintainer": "Schievel",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "features": {
+        "encoder": true,
+        "oled": true,
+        "rgb_matrix": true
+    },
+    "build": {
+        "debounce_type" : "asym_eager_defer_pk"
+    },
+    "usb": {
+        "shared_endpoint": {
+            "keyboard": true
+        }
+    },
+    "matrix_pins": {
+        "rows": ["GP4", "GP5", "GP6", "GP7", "GP8", "GP9"],
+        "cols": ["GP10", "GP11", "GP12", "GP13", "GP14", "GP15"]
+    },
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP27", "pin_b": "GP26"}
+        ]
+    },
+    "rgb_matrix": {
+        "driver": "ws2812",
+        "layout": [
+            {"matrix": [3, 5], "x": 85, "y": 24, "flags": 1},
```

Comment:
These aren't all modifiers.  Most of these are alpha keys, and the flags should be updated to represent that